gateway-service-admin/pom.xml

```diff
@@ -90,6 +90,14 @@
             <groupId>com.sun.xml.bind</groupId>
             <artifactId>jaxb-impl</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Do we need a full commons-lang3 dependency just to check if a string is blank? Seems overkill.

Reply:
Well, you are right, I prefer all the StringUtils handling of nulls, but we can do with 
`value != null && ! value.isEmpty()`